Stock discovery screen revamp

beckskelly opened the pull request:
Its the weekend so theres no need to review this until Monday.

Changed up the stock discovery screen as per Andreas comments. I basically widened the containers to fit the entire screen with padding on either end.  I tried changing the background colour but I felt it looked a bit dirty, so I left it as is for now, but can always change it to a background color with no border on the cards if you think its bettter

Let me know what you think.
![image](https://user-images.githubusercontent.com/78801723/204048176-f80f9ad3-ee9a-43c3-9436-6eb5aa16eb96.png)
![image](https://user-images.githubusercontent.com/78801723/204048385-8f0684bb-3e3f-4d4e-9d91-78204c974cbb.png)


Attempt with background color changed:
![image](https://user-images.githubusercontent.com/78801723/204044224-7515599f-d340-44cb-a03b-4925eeb7206f.png)

WarrenK-design commented on src/components/stockDiscoveryComponents/stockSearchResults/StockSearchResults.js:
```diff
@@ -37,8 +37,9 @@ function StockSearchResults({keyword}) {
         {loading ? <LoadingSpinner /> : error  ? <MessageAlert variant='danger'>{error}</MessageAlert> :
         Object.keys(stocks).length !== 0  ?
         <>
+        <div className="stocksearchresults">
```
Again have a look into Flexbox using containers, I see you have replaced Container with a div here and styling has effected the search bar. 

## Styling as Div 
When code is ```div className="stocksearchresults"```
![image](https://user-images.githubusercontent.com/61060096/204103519-fb2da12f-0d7a-49b2-a9ff-33fe25a6c155.png)

## Styling as Container 
When code is just ```<Container> ``` no class. You can use ```<div>``` if you want but just might be handier if bootsrap fixes these issues for you. 
![image](https://user-images.githubusercontent.com/61060096/204103653-89c4c657-7199-4167-9c39-b634f95d9d8c.png)

WarrenK-design reviewed (changes requested):
Hi Becks, 

Looks good though I think if we aren't changing the background colour of the cards or the screen itself the border needs to be more defined on the cards. 

![image](https://user-images.githubusercontent.com/61060096/204102746-ae8d0b7f-1b89-4278-992d-39f3c8040a1e.png)

Is a little overwhelming on some screens but ah that's only small thing.  
![image](https://user-images.githubusercontent.com/61060096/204102758-04d831f2-ad1c-4bc5-b86d-a92434fecaf7.png)

Netflix get away with using images for borders as backgrounds but images bit tricky and time consuming, think the black border will do. 
![image](https://user-images.githubusercontent.com/61060096/204102809-28db928c-708c-4fe5-8d47-d94f87b8fc7c.png)

```
.tickercardstyle {
  width: 10rem;
  height: 100%;
  display: block;
  border-width: 0.175rem;
  border-color: black;
}
```

Also getting this warning. 
![image](https://user-images.githubusercontent.com/61060096/204103058-5fac7706-8194-4d5a-867a-ec3a33b94110.png)

beckskelly commented:
Cheers for the review @WarrenK-design especially over the weekend. I think the black is a bit intense, but I'm sure there can be a workaround, maybe a shadow or something. I know what you mean about the container, I didn't use one on purpose to get the div to fit the width of the screen, but I might give it a go just altering the container, as you pointed out, bootstrap works best with its own elements

beckskelly commented:
Ok so I made the following changes:

1. Added a container around the search results, fixing the styling issue on searchbar
2. Left the div as is for the summary page, containers don't like going 100% width
3. Made the border of the cards slightly fatter, darker, and added a shadow

Let me know what you think!
![image](https://user-images.githubusercontent.com/78801723/204261386-15de077b-c53a-421e-aeab-0aa624c7d465.png)

WarrenK-design reviewed (approved):
Hi Becks looks good, the shadows definitely better than the black border. 

Might be something to add to the spreadsheet but I dont mind doing it nearly have code but not priority but when I do a search for a large search like "a". This sidescroll is bit annoying./lot of white space. 

![image](https://user-images.githubusercontent.com/61060096/204269594-bf64233c-4878-416b-b763-de0ee2d2acb9.png)


Id rather something like this . 
![image](https://user-images.githubusercontent.com/61060096/204269284-06af5439-931c-45c1-bc9c-263269bc44f1.png)

![image](https://user-images.githubusercontent.com/61060096/204269332-fe0d96fb-5433-4717-afdc-c0b03c3966b3.png)

Again bit like Netflix 
![image](https://user-images.githubusercontent.com/61060096/204269426-f6dfeefe-b841-4e07-afc1-a2d1e28729ae.png)

joeycor92 commented:
Looks great after these changes. Thanks very much for all of this!
